compute_sdk/globus_compute_sdk/sdk/utils/sample_function.py

```diff
@@ -0,0 +1,11 @@
+def sdk_tutorial_sample_function():
+    """
+    Simple echo function in its own module for diagnostic tests
+
+    Note that this function still may not deserialize with some
+    serialization methods such as DillCode due to module import issues
+    """
+    from datetime import datetime as dt
+    from datetime import timezone
+
+    return f"Hello world at {dt.now(timezone.utc).isoformat()}"
```

Comment:
I think the utility of this is to ensure that the simplest possible function can run, so consider dropping the imports and only returning `Hello world`.

Reply:
I added the import in there as ensuring that imports work in serialization is also a test.  Can making it simpler to test less things.  Perhaps I'll add a separate simple hello world and leave this one as an optional one...

Reply:
I vote for one function without imports, but not a blocker.

Reply:
I now use the simple hello world function as default.